RoddyCore/test/de/dkfz/roddy/core/ExecutionContextTest.groovy

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2017 eilslabs.
+ *
+ * Distributed under the MIT License (license terms are at https://www.github.com/eilslabs/Roddy/LICENSE.txt).
+ */
+
+package de.dkfz.roddy.core
+
+import de.dkfz.roddy.RunMode
+import de.dkfz.roddy.execution.io.ExecutionService
+import de.dkfz.roddy.execution.io.LocalExecutionService
+import de.dkfz.roddy.execution.io.fs.FileSystemAccessProvider
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by heinold on 01.02.17.
+ */
+public class ExecutionContextTest {
```

Comment:
Do I see this right, that these methods are only part of the ExecutionContext, because they need it for error reporting? If so, would it not be more appropriate to just put it at the right place directly, and fix their interface, as soon as we have the error reporting refactored?